gtsfm/two_view_estimator.py

```diff
@@ -152,23 +154,37 @@ def create_computation_graph(
             i2Ui1_pp_graph,
             v_corr_idxs_pp_graph,
             two_view_report_pp_graph,
-        ) = self.processor.create_computation_graph(
-            i2Ri1_graph, i2Ui1_graph, v_corr_idxs_graph, two_view_report_graph
-        )
+        ) = self.processor.create_computation_graph(i2Ri1_graph, i2Ui1_graph, v_corr_idxs_graph, two_view_report_graph)
         # We provide both, as we will create reports for both.
         return (i2Ri1_pp_graph, i2Ui1_pp_graph, v_corr_idxs_pp_graph, two_view_report_graph, two_view_report_pp_graph)
 
 
 def generate_two_view_report(
     inlier_ratio_est_model: float,
-    R_error_deg: float,
-    U_error_deg: float,
-    num_inliers_gt_model: int,
-    inlier_ratio_gt_model: float,
-    v_corr_idxs_inlier_mask_gt: np.ndarray,
     v_corr_idxs: np.ndarray,
+    R_error_deg: Optional[float] = None,
+    U_error_deg: Optional[float] = None,
+    v_corr_idxs_inlier_mask_gt: Optional[np.ndarray] = None,
+    reproj_error_gt_model: Optional[np.ndarray] = None,
 ) -> TwoViewEstimationReport:
     """Wrapper around class constructor for Dask."""
+    # Compute ground truth metrics.
+    if v_corr_idxs_inlier_mask_gt is not None and reproj_error_gt_model is not None:
+        num_inliers_gt_model = np.count_nonzero(v_corr_idxs_inlier_mask_gt)
+        inlier_ratio_gt_model = (
+            np.count_nonzero(v_corr_idxs_inlier_mask_gt) / v_corr_idxs.shape[0] if len(v_corr_idxs) > 0 else 0.0
+        )
```

Comment:
Yeah I'm getting a lot of mypy errors for a lot of variables because many of them are labeled as `Optional`. I think this should be part of a larger effort where we reassess whether certain variables are, in fact, optional.